test/es-module/test-esm-import-meta-main-eval.mjs

```diff
@@ -0,0 +1,37 @@
+import { escapePOSIXShell, mustSucceed } from '../common/index.mjs';
+import child from 'node:child_process';
+
+const execOptions = '--input-type module';
+
+const script = `
+import { mustCall } from '../common/index.mjs';
+import assert from 'node:assert/strict';
+
+assert.strictEqual(import.meta.main, true);
+
+import('../fixtures/es-modules/import-meta-main.mjs').then(
```

Comment:
```suggestion
await import(${JSON.stringify(fixtures.fileURL('es-modules/import-meta-main.mjs'))}).then(
```

Reply:
Are you confident that this does not break the unua

Reply:
Yes I'm confident, and that removes the need to pass a `cwd` or to refactor the test if we ever move the test file or the fixtures dir.